### ISSUE TITLE
Still work ?

### ISSUE THREAD

**Issue by ilkkaya** [twink0r/nntp-proxy#1]
Does it work?

**Comment by twink0r** [comment]
Yes it should work! Sometimes we got trouble with connections will not reset.

**Comment by ilkkaya** [comment]
Thanks for info, i will try. can you will make wiki ?

**Comment by ilkkaya** [comment]
2018/03/23 21:58:13 [PLAIN - DO NOT USE PROD!] Listening on 0.0.0.0:3333
2018/03/23 21:58:13 [Dispatch] Command : authinfo user 1
2018/03/23 21:58:13 [Dispatch] Command : authinfo user 1
2018/03/23 21:58:13 [CONN] Error dropping Backend Connection cause selectedBacke          nd is nil
2018/03/23 21:58:13 &{0x1187a588 <nil> authinfo user 1 <nil>}
2018/03/23 21:58:13 [CONN] Error dropping Backend Connection cause selectedBacke          nd is nil
2018/03/23 21:58:13 &{0x1187a598 <nil> authinfo user 1 <nil>}
2018/03/23 21:58:17 [Dispatch] Command : authinfo user 1
2018/03/23 21:58:17 [Dispatch] Command : authinfo user 1
2018/03/23 21:58:17 [CONN] Error dropping Backend Connection cause selectedBacke          nd is nil
